src/service/gateway.proto

```diff
@@ -13,8 +14,25 @@ enum close_state {
   close_state_dispute = 3;
 }
 
+enum error_code {
+  error_generic = 0;
+  error_invalid_signature = 1;
+  error_not_asserted = 2;
+  error_unknown_region = 3;
+}
+
 /* general */
 
+message gateway_success_resp {
```

Comment:
not needed as part of this PR right? If so, what are they actually useful for? Generic "bytes" on success is quite vague?

Reply:
success resp is not required but the error resp is required and these two go together.

Reply:
success_resp is used for RPCs where there is no defined response payload to be returned.  For example when submitting a POC witness or receipt report.  In that example 'resp' has a value of ok....prob does make sense to do something else there than bytes

Reply:
how about just an empty success response?

Reply:
yeah prob as much as it needs.  will roll with that